script/Ciel/Day 7/07_01_CIEL07_2_1A.txt

```diff
@@ -78,60 +80,62 @@ Senpai doesn't seem to appreciate what happened between me and Noel.
 -- Page 176, Offset 15226.
 -- ……そうですね。座ってもぜんぜん落ち着かないのは疑問があるからです。いい機会なのではっきりさせましょう」
 How do I put it... I guess what mystifies me is that even though you sat down I'm still bothered.
- We might as well take the opportunity to clear something up for me."
+ I suppose I may as well take this opportunity to clear something up."
 }
 [sha:c97365d70967fbd82d50e9ec9f4e236c827d3ac6]{
 -- Page 177, Offset 15227.
 -- シエル先輩はやや前屈みになって俺の顔を上目遣いに覗きこむ。
-Ciel bends forward slightly, peering into my face from below.
+Ciel leans forward slightly, and peers up at my face from below.
 }
 [sha:509906bb152218d63bb8c5ae49ac52917f7e450a]{
 -- Page 178, Offset 15228.
 -- 「自分から呼んでおいて今更ですけど。
 // kinda long, because of 今更
-"You might feel its a bit late asking this after I told you to come here,
+"This may sound like a strange thing to ask at this point, seeing as how I told you to come here, but―――
 }
 [sha:2a0b98697042fdf9db53bb76de5a668990c79014]{
 -- Page 178, Offset 15229.
 -- なぜノエル先生ではなく、わたしに構うんですか？」
-but why are so concerned with me, rather than Noel-sensei?"
+Why is it that you want to spend time with me, rather than Noel-sensei?"
```

Comment:
Putting it this way sounds like she's asking about the situation right now rather than their relationship in general.